python_ta/contracts/__init__.py

```diff
@@ -259,6 +259,13 @@ def _check_function_contracts(wrapped, instance, args, kwargs):
     return r
 
 
+def check_type_strict(argname: str, value, expected_type) -> None:
```

Comment:
Please put in type annotations for `value` (`Any`) and `expected_type` (`type`).